src/Broadway/EventStore/EventStoreInterface.php

```diff
@@ -23,11 +23,12 @@
      *
      * @return DomainEventStreamInterface
      */
-    public function load($id);
+    public function load($streamType, $identifier);
```

Comment:
`$streamType` :+1:  or just `$type` (you already know you're getting a `DomainEventStreamInterface`